foyer/AST2NetworkX.py

```diff
@@ -0,0 +1,190 @@
+from networkx.algorithms import isomorphism
+from foyer.smarts import SMARTS
+from plyplus.strees import STree
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+class AST2NetworkX:
```

Comment:
You might want to consider subclassing networkx.Graph, so that your instances can be used everywhere where networkX graphs can be used (leveraging duck typing).